src/test/end-to-end/tests/login-test.ts

```diff
@@ -117,9 +117,41 @@ Scenario('I as a media user should be able to see the beta tag and feedback link
     I.seeBetaFeedbackOnPage('b2c/reset-pw');
 });
 
+Scenario('I as a CFT user should be able to sign-in with the valid credentials', async ({I}) => {
+    I.loginAsCftUser();
+    I.waitForText('Your account');
+    I.logout();
+}).tag('@CrossBrowser');
+
+Scenario(
+    'I as a CFT user should be able to see proper error messages when username or password fields are empty',
+    async ({I}) => {
+        I.loginAsCftUser('', '');
+        I.waitForText('Email address cannot be blank');
+        I.see('Password cannot be blank');
+    }
+).tag('@Nightly');
+
+Scenario(
+    'I as a CFT user should be able to see proper error message when username or password is wrong',
+    async ({I}) => {
+        I.loginAsCftUser('email@justice.gov.uk', 'password');
+        I.waitForText('Incorrect email or password');
+    }
+).tag('@Nightly');
+
+Scenario(
```

Comment:
We will also need a test for an invalid CFT account I think - where they have an account but don't have the correct role

Reply:
I will add it :)